src/controller/discoverymgr/mnedc/client/client.go

```diff
@@ -98,6 +97,11 @@ type MNEDCClient interface {
 	SetClient(clientAPI restclient.Clienter)
 }
 
+type conf struct {
+	ServerIp string `yaml:"server-ip"`
```

Comment:
How about 'ServerIP'?

Reply:
Sure, I would change it. 'ServerIP' looks better.